azure-pipelines.yml

```diff
@@ -1,7 +1,4 @@
 ---
-trigger:
```

Comment:
Right, I forgot to mention that this would have been to be removed to build all branches, sorry for the omission—and thanks for merging the patch series :smile: